.github/workflows/release-python.yaml

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Publish Python
```

Comment:
This seems not triggered on tags, here is a pipeline triggered by me: https://github.com/apache/fory/actions/runs/17008234471

Reply:
I think it needs to be merged first:

> This event will only trigger a workflow run if the workflow file exists on the default branch.

https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#workflow_run